sdk/nodejs/output.ts

```diff
@@ -534,6 +534,20 @@ export function secret<T>(val: Input<T | undefined>): Output<Unwrap<T | undefine
         o.isKnown, Promise.resolve(true), o.allResources!());
 }
 
+/**
+ * [unsecret] behaves the same as [output] except the returned output takes the existing output and unwraps the secret
+ */
+export function unsecret<T>(val: Output<T>): Output<T> {
+   return new Output(
+        val.resources(), val.promise(/*withUnknowns*/ true),
+        val.isKnown, Promise.resolve(false), val.allResources!());
+}
+
+export function isSecret<T>(val: Output<T>): Promise<boolean> {
```

Comment:
Similarly, should we consider making this accept `Input<T>` (and if it's not an `Output`, it'd return `false`)?